constructor/post_install.bat

```diff
@@ -1,2 +1,3 @@
 "%PREFIX%\Scripts\activate.bat" "%PREFIX%"
 robotlab-shortcuts
+pip install robotframework-whitelibrary

```

Comment:
I've been meaning to explore packaging it. Worst case is we'll have a `_robotlab_wheels_` package or something, and in post-install, do:
```bat
%PREFIX%\Scripts\activate  %PREFIX%
python -m pip install --no-index --find-links %PREFIX%/share/wheelhouse robotframework-whitelibrary
```

I'll give that a shot, and will separately look into "proper" conda packaging of it (e.g. build from source, separate package for the c part, etc.)

Reply:
:link: #30